src/gui/search/searchsortmodel.h

```diff
@@ -30,10 +30,12 @@
 #define SEARCHSORTMODEL_H
 
 #include <QSortFilterProxyModel>
+#include <QStringList>
 #include "base/utils/string.h"
 
 class SearchSortModel: public QSortFilterProxyModel
 {
+    using base = QSortFilterProxyModel;
```

Comment:
>Think of it as a kind of the language keyword?

I've seen this somewhere... C# or Java (or something like that). I won't focus on this.
>You correct one type alias and voilà, all base virtual function calls are correct

...unless new base class has the same functions :)
Okay, let it be. Anyway all is not provided.